stylesheets/_component.lexicon.scss

```diff
@@ -0,0 +1,22 @@
+.example {
+    .wrapper {
```

Comment:
Could you add some comments here to detail which example UI the styles are for?

Reply:
Will rework that in ticket #641

Reply:
Would prefer it if we scoped class names that were specifically for the lexicon, `.lex-something` for example.

Reply:
Also, is there any need to nest the selector here, `.example .wrapper` seems cleaner. Surprised the linter didn't flag this, though that might not be enabled yet.